source/tamasha/ui/content.tid

```diff
@@ -1,10 +1,8 @@
 created: 20201231053912984
-modified: 20210126142114825
+modified: 20210128205353357
 title: $:/plugins/kookma/tamasha/ui/content
 type: text/vnd.tiddlywiki
 
-\define contentStyle() width:$(contentWidth)$;height:$(contentHeight)$;
-\define contentClass() tamasha-slide-content $(myTheme)$
```

Comment:
@kookma by keeping th etheme, did you mean this line? I think I removed this by mistake.

Reply:
Yes, Theme is applied at two points: one in content $:/plugins/kookma/tamasha/ui/content

```
\define contentClass() tamasha-slide-content $(myTheme)$
```

and the next in presenter

```
<div class=<<myTheme>> ><!-- apply theme -->
```